src/OneExplorer/OneExplorer.ts

```diff
@@ -500,9 +499,18 @@ input_path=${modelName}.${extName}
 
 export function initOneExplorer(context: vscode.ExtensionContext) {
   // TODO Support multi-root workspace
-  let workspaceRoot: vscode.Uri = vscode.Uri.file(obtainWorkspaceRoot());
-  // NOTE: Fix `obtainWorksapceRoot` if non-null assertion is false
-  const oneTreeDataProvider = new OneTreeDataProvider(workspaceRoot!);
+  let workspaceRoot: vscode.Uri|undefined = undefined;
+  try {
+    workspaceRoot = vscode.Uri.file(obtainWorkspaceRoot());
+  } catch (e: unknown) {
+    if (e instanceof Error) {
+      Logger.error('OneExplorer', e.message);
+    } else {
+      throw e;
```

Comment:
(Q) What case does this if/else syntax distinguish?

If you throw an error, ToolchainView does not work. Is it your intention?